### PR TITLE
Resolve merge

### PR DESCRIPTION
**Pull request by solijonovam**
resolves the PR comments from issue 262 
also, resolves the merge conflict

### CODE REVIEW

File: app/static/js/volunteerProfile.js [hunk 1]
```diff
@@ -113,3 +113,19 @@ $(document).ready(function(){
   });
 
 });
+
+function updateManagers(el, volunteer_username, action ){// retrieve the data of the studnet staff and program id if the boxes are checked or not
+  var program_id=$(el).attr('data-programid');
+  console.log(volunteer_username)
+  console.log(program_id)
+  console.log(action)
```
Comment (BrianRamsay) [inline app/static/js/volunteerProfile.js]: Remove the debug output

**Review by BrianRamsay — CHANGES_REQUESTED** [review]

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
We should have an entry in AdminLog for this action. It can go in the controller.

**Review by BrianRamsay — DISMISSED** [review]

**Comment by BrianRamsay** [comment]
Looks good! But there's a conflict to resolve before I can merge it.

**Comment by BrianRamsay** [comment]
Also, let's switch the positions on the page of the Service Learning Faculty table and the Course Proposal Management table.

**Comment by solijonovam** [comment]
your comment seems to refer to another issue that is not related to the current PR.

**Comment by BrianRamsay** [comment]
When you resolved the merge, you kept some of the wrong changes. You'll need to reset the branch back to the commit before the merge and try again (or create a new branch from that commit to preserve this one)

**Comment by BrianRamsay** [comment]
Do you see how the tests fail? That's how I know you're not resolving the merge correctly. The `StudentManager` table you are trying to use no longer exists.

We can talk about it this afternoon if we need to.

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**